resources/src/books.php

```diff
@@ -125,9 +124,9 @@ public function add()
           </div>';
           if (isset($_POST["submit"]))
          {
-            $dest_path=$ob->config["paths"]["images"].$_FILES['bcover']['name'];
+            $dest_path=$this->ob->config["paths"]["images"].$_FILES['bcover']['name'];
```

Comment:
This might require to  add a `/` between `config["paths"]["images"]` and `$_FILES['bcover']['name']`

Reply:
can you show how exactly you mean?
mmm something like this ->
```php
$dest_path=$this->ob->config["paths"]["images"] ."/" .$_FILES['bcover']['name']
```

Reply:
Yes

Reply:
ok hold on